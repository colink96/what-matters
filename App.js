import React from 'react';
import { StyleSheet, Text, View } from 'react-native';
import { Provider } from 'react-redux';

import store from './store';
import AllTasks from './components/allTasks';

export default function App() {
  return (
    <Provider store={store}>
      <View style={styles.container}>
        <AllTasks />
      </View>
    </Provider>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    alignItems: 'center',
    justifyContent: 'center',
  },
});
