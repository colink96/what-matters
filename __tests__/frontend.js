import React from 'react';
import App from '../App';

import renderer from 'react-test-renderer';

test('renders correctly', async () => {
  let tree;
  renderer.act(() => {
    tree = renderer.create(<App />);
  });
  expect(tree.toJSON()).toMatchSnapshot();
});
