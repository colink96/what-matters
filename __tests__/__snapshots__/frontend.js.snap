// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#454ADE",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "white",
        "borderRadius": 8,
        "flex": 1,
        "marginHorizontal": 10,
        "marginVertical": 40,
        "padding": 20,
      }
    }
  >
    <View
      style={
        Object {
          "borderRadius": 8,
          "flexDirection": "row",
          "justifyContent": "space-between",
        }
      }
    >
      <View
        style={
          Object {
            "alignSelf": "center",
          }
        }
      >
        <Text
          style={
            Object {
              "fontFamily": "Verdana",
              "fontSize": 20,
            }
          }
        >
          Saturday
          ,
           
          April
           
          11
          ,
           
          2020
        </Text>
      </View>
      <View
        accessible={true}
        focusable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignContent": "center",
            "backgroundColor": "#454ADE",
            "borderRadius": 25,
            "flexDirection": "row",
            "height": 50,
            "justifyContent": "center",
            "marginBottom": 5,
            "marginTop": 5,
            "paddingBottom": 5,
            "paddingTop": 5,
            "width": 50,
          }
        }
      >
        <View
          style={null}
        >
          <Text
            style={
              Object {
                "color": "white",
                "fontSize": 30,
              }
            }
          >
            +
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "column",
          "justifyContent": "flex-start",
        }
      }
    >
      <Text
        style={
          Object {
            "alignSelf": "flex-start",
            "color": "#347FC4",
            "fontFamily": "Verdana-Bold",
            "marginBottom": 40,
            "marginTop": 10,
          }
        }
      >
        0
         tasks
      </Text>
    </View>
  </View>
</View>
`;
